generate.py

```diff
@@ -172,7 +172,7 @@ def main():
         outputs.extend(net.push_ops)
 
         print('Priming generation...')
-        for i, x in enumerate(waveform[:-(args.window + 1)]):
+        for i, x in enumerate(waveform[-args.window: -1]):
```

Comment:
So instead of priming with the first args.window samples from the seed waveform, this takes the last args.window samples. Why does that make any difference?